konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
@TomasHo `<title>` is not suppose to be displayed on the page. It used to display text in your browser's tab/window. If you want to show the text on the page you need to add something like `<h1>{% block title %} {% endblock %}</h1>` to `base.html` to the `<body>`.